ndb/src/google/cloud/ndb/model.py

```diff
@@ -2047,6 +2047,21 @@ def __init__(self, *args, **kwargs):
 
         super(TextProperty, self).__init__(*args, **kwargs)
 
+    def _constructor_info(self):
```

Comment:
This "snuck" in because I realized that `repr()` was broken for `TextProperty` and `StringProperty` when testing `MetaModel.__repr__`